hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/diskbalancer/command/QueryCommand.java

```diff
@@ -84,7 +84,7 @@ public void execute(CommandLine cmd) throws Exception {
         System.out.printf("%s", workStatus.currentStateString());
       }
     } catch (DiskBalancerException ex) {
-      LOG.error("Query plan failed. ex: {}", ex);
+      LOG.error("Query plan failed. ex: {}", ex.getMessage());
```

Comment:
How about we print the entire stacktrace and remove placeholder from log line?

Reply:
Thanks @virajjasani for your review. I agree with you and I will update it.

Reply:
In general, it would depend on how useful the message is. I believe if we are re-throwing Exceptions (like this case), and we are also logging the error message at ERROR level, it's good to print the entire stracktrace for better debugging. Sounds good?

Reply:
Thanks @virajjasani for your comments. I checked several other places, all specifying `message:{}`. So I'll just change this one.